src/psyclone/tests/dynamo0p3_stubgen_test.py

```diff
@@ -823,15 +823,15 @@ def test_stub_stencil_extent():
     generated_code = str(kernel.gen_stub)
     result1 = (
         "SUBROUTINE testkern_stencil_code(nlayers, field_1_w1, "
-        "field_2_w2, field_2_stencil_size, field_2_stencil_dofmap, "
+        "field_2_w2, field_2_stencil_sizes, field_2_stencil_dofmap, "
         "field_3_w2, field_4_w3, ndf_w1, undf_w1, map_w1, ndf_w2, "
         "undf_w2, map_w2, ndf_w3, undf_w3, map_w3)")
     assert result1 in generated_code
-    result2 = "INTEGER(KIND=i_def), intent(in) :: field_2_stencil_size"
+    result2 = "INTEGER(KIND=i_def), intent(in) :: field_2_stencil_sizes"
```

Comment:
It's a bit weird that this is now named "...sizes" plural but is still a single integer value. Is it possible to put this back?

Reply:
Unless I'm missing something this has to have the same name as the variables generated in the PSylayer as the name is pulled from the same methods. I could revert everything back to being ???_stencil_size. My original reason for changing the name was that it made more sense in the context of the variable in the PSylayer where they are now arrays of integers. I agree that in the kernels it may be more confusing and that is what the scientists see so.... Thoughts?

Reply:
I've changed everything back to ???_stencil_size.